docs/readthedocs/conf.py

```diff
@@ -18,12 +18,12 @@
 # -- Project information -----------------------------------------------------
 
 project = 'PowerSystemDataModel'
-copyright = u'2020. TU Dortmund University, Institute of Energy Systems, Energy Efficiency and Energy Economics, Research group Distribution grid planning and operation '
+copyright = u'2023. TU Dortmund University, Institute of Energy Systems, Energy Efficiency and Energy Economics, Research group Distribution grid planning and operation '
 author = 'Johannes Hiry, Debopama Sen Sarma, Chris Kittl'
```

Comment:
This could be set to `ie³` since we do not want to adapt this String all the time